bin/create-initial-roles-and-users.sh

```diff
@@ -67,3 +67,4 @@ cd "$parent_path"
 ./idam-create-caseworker.sh caseworker,caseworker-caa,caseworker-befta_master,caseworker-befta_jurisdiction_1,caseworker-befta_jurisdiction_2 befta.caseworker.caa@gmail.com Pa55word11
 ./idam-create-caseworker.sh caseworker,caseworker-befta_master,caseworker-befta_master-solicitor befta_master.solicitor.becky@gmail.com Pa55word11 Solicington Becky
 ./idam-create-caseworker.sh caseworker,caseworker-befta_master,caseworker-befta_master-solicitor befta_master.solicitor.benjamin@gmail.com Pa55word11 Solicington Benjamin
+./idam-create-caseworker.sh caseworker,caseworker-caa befta.caseworker.caa@gmail.com Pa55word11 CAA Caseworker
```

Comment:
befta.caseworker.caa@gmail.com already exists in this file.

Reply:
When running data-store FTA against IDAM built with this script: I get 403 errors from the first 3 scenarios in  'F-103: Get Case-Assigned Users and Roles'.  

They all fail running ' a successful call [by Jamal to assign Dil a few case roles to access C1] as in [F-103_Jamal_Assign_Dil_Case_Role_To_C1]'.

Is this because the CAA account should remain a member of the jurisdictions that it was previously configured with or is the FTA script incorrectly using the CAA account for this call?

Reply:
The FTAs should use a different user. CAA will have only caseworker and caseworker-caa roles.